leap/leap.py

```diff
@@ -1,2 +1,15 @@
 def leap_year(year):
-    pass
+    is_leap = 0
+    if year%4==0:
+        if year%100!=0:
+            is_leap = 1
+        else:
+            if year%400==0:
```

Comment:
This seems redundant. If the year is divisible by 400 then it is also divisible by 4 and 100. Did you have a reason or a test case that requires this line? There could be one but I can't think of one.

Reply:
The description of a leap year is the following:
evenly divisible by 4, but not by 100, unless is evenly divisible by 400. 
How can get rid of this last constraint?